cinema/serializers.py

```diff
@@ -135,3 +136,43 @@ def create(self, validated_data):
 
 class OrderListSerializer(OrderSerializer):
     tickets = TicketListSerializer(many=True, read_only=True)
+
+
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ['id', 'image']
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(source='image.url', read_only=True)
+
+    class Meta:
+        model = Movie
+        fields = ['id', 'title', 'description', 'duration', 'image']
```

Comment:
The `MovieSerializer` is defined twice in the file, which can lead to confusion and potential errors. Ensure that each serializer class has a unique name and is defined only once.